api/tests/mocha/services/generate-xform.spec.js

```diff
@@ -247,6 +248,14 @@ describe('generate-xform service', () => {
   });
 
   describe('update', () => {
+    let generate;
+    let unset;
+
+    beforeEach(() => {
+      generate = sinon.stub();
+      unset = service.__set__('generate', generate);
```

Comment:
Took me a bit to figure out why I had done this. Turns out, for some reason I had updated [this line](https://github.com/medic/cht-core/blob/6e080fcca0abb96d8d4a7aa5255fdddfbee6b6e2/api/src/services/generate-xform.js#L199) from 
```.js
return module.exports.generate(...
```
to just be 

```.js
return generate(...
```

Might have been because of the diff from the old PR, or it just might have been me thinking it should be cleaned up (no need to reference `exports` when we can just call directly).  Either way, this caused the tests to fail in a way that I did not understand and after a bunch of fiddling, I thought I needed `rewire` to fix things (mostly because I did not really understand module exports very well at the time and did not realize the test errors were related to that change...).

Anyway, TLDR is that I am rolling back both changes.